style.css

```diff
@@ -1,57 +1,64 @@
-body {
-  background-color: #d1c0c0;
-  font-family: 'Staatliches', cursive;
-  margin: auto;
-  width: 50%;
-  text-align: center
-}
-
-body div {
-  color: #423f3c;
-}
-
-.title {
-}
-
-.name {
-  font-size: 750%;
-  margin-bottom: 0;
-}
-
-.subname {
-  font-size: 500%;
-  margin-top: 0;
-  text-align: right;
-}
-
-.calendar {
-  margin-bottom: 50px;
-}
-
-.calendar small {
-  font-size: 50%;
-}
-
-.calendar p {
-  margin-bottom: 0;
-}
-
-.event-primary {
-  font-size: 400%;
-}
-
-.event-secondary {
-  font-size: 300%;
-}
-
-.links {
-  display: flex;
-  justify-content: space-around;
-}
-
-.links a {
-  font-size: 250%;
-  margin-right: 10%;
-  margin-left: 10%;
-  color: #ed8c72;
-}
+body {
+  background-color: #d1c0c0;
+  font-family: 'Staatliches', cursive;
+  margin: auto;
+  width: 50%;
+  text-align: center
+}
+
+body div {
+  color: #423f3c;
+}
+
+li {
+    list-style: none;
+}
+
+.title {
+}
+
+.name {
+  font-size: 750%;
+  margin-bottom: 0;
+}
+
+.subname {
+  font-size: 500%;
+  margin-top: 0;
+  text-align: right;
+}
+
+.calendar {
+  margin-bottom: 50px;
+}
+
+.calendar small {
+  font-size: 50%;
+}
+
+.calendar p {
+  margin-bottom: 0;
+}
+
+.event {
+    display: block;

```

Comment:
Please use two spaces, not four, to match the formatting of the rest of the code.